partials/language-specific-deploy/haskell.md

```diff
@@ -31,32 +32,36 @@ main = scotty 8080 $ do
 
 ### Dependencies
 
-Make sure to list all your dependencies in your `.cabal` file. For the example above, you need:
+Make sure to list all your dependencies in your `.cabal` file.
 
-```
-  build-depends:       base
-                     , scotty
+For the example above, you need:
+
+```cabal
+executable myfirstapp
+    main-is: Main.hs
+    build-depends: base
+                 , scotty
 ```
 
 ### Cached dependencies
 
-#### Enabling dependencies caching
+You can enable dependencies caching by adding the `CC_CACHE_DEPENDENCIES=true`.
+
+It is enabled by default only for rust and haskell applications.
 
-You can enable dependencies caching by adding the `CC_CACHE_DEPENDENCIES=true` [environment variable](#setting-up-environment-variables-on-clever-cloud) in your application. It is enabled by default only for rust and haskell applications.
+To disable dependencies caching completely by removing the `CC_CACHE_DEPENDENCIES` environment variable or by setting it to `false`.
 
-#### Disabling dependencies caching
+To fully remove cached dependencies, you have to rebuild your application from scratch.
```

Comment:
Are you sure dropping the CC_CACHE_DEPENDENCIES variable isn't enough?

Reply:
Well I am not sure, I am just confident with a rebuild and restart which make sure that it will drop the cache.

Reply:
Rebuild + restart won't "remove" the cache, the deployment will just ignore downloading it. But the next deployment will still use the cached dependencies if `CC_CACHE_DEPENDENCIES=true`.

I think it may be better to tell users to set `CC_CACHE_DEPENDENCIES=false` to completely disable the cache and use "rebuild + restart" if they need to not use the cache for one deployment only.